repository_service_tuf_worker/repository.py

```diff
@@ -355,9 +363,23 @@ def _update_state(
                 if len(targets) == 0:
                     logging.debug(f"Update: {role_name} completed")
                     completed_roles.append(role_name)
+            if subtask.status == states.FAILURE:
+                exc_type = subtask.result.__class__.__name__
+                exc_message = list(subtask.result.args)
+                _update_state(
+                    states.FAILURE,
+                    bin_targets,
+                    completed_roles,
+                    exc_type=exc_type,
+                    exc_message=exc_message,
+                )
+                raise ChordError(
+                    f"Failed to execute {subtask.task_id}: "
+                    f"{exc_type} {exc_message}"
+                )
 
             if sorted(completed_roles) != sorted(list(bin_targets)):
-                _update_state(bin_targets, completed_roles)
+                _update_state("RUNNING", bin_targets, completed_roles)
```

Comment:
I specifically opened the `Celery.result.states` module and I don't find a state called `RUNNING`.
Probably here it could be a custom string, but I just wanted to mention that.

Reply:
Yes, there is no running :(

Reply:
This means that here we could have a custom string as a state, right?
This is correct?

Reply:
Yes, we have a custom string as state RUNNING.
I notice that we declare them here, andt it can make it more consistent.

https://github.com/vmware/repository-service-tuf-worker/blob/fda0e273e974acb6c957d35b4dfe9a43827c9654/app.py#L30-L36

I will not add it to this PR. Otherwise, it will change the focus and the primary goal.

The new issue is:

Make the state more consistent.

1. Rename from `status` to `STATE` as a constant
https://github.com/vmware/repository-service-tuf-worker/blob/fda0e273e974acb6c957d35b4dfe9a43827c9654/app.py#L30
2. Move it to `repository_service_tuf_worker.__init__`
3. Use the Celery `states` instead new strings, keep the strings only for RSTUF custom
```python
 class STATE(Enum): 
     RECEIVED = states.RECEIVED 
     PRE_RUN = "PRE_RUN" 
     RUNNING = "RUNNING" 
     SUCCESS = states.SUCCESS
     UNKNOWN = "UNKNOWN" 
     FAILURE = states.FAILURE
``` 
4. Apply the refactoring to all usage (including UT)

Reply:
Issue created